library/Garp/Service/Amazon/Ses.php

```diff
@@ -153,163 +153,6 @@ public function listVerifiedEmailAddresses()
         return $response->get('VerifiedEmailAddresses');
     }
 
```

Comment:
In plaats van de methode alleen te verwijderen, is het misschien goed om de methode te laten staan en een deprecation notice te triggeren?

Reply:
Ik heb het weggegooid omdat het alternatief niet ingewikkeld is. Dit kan vrij makkelijk omgeschreven worden naar `Garp_Mailer`. Maar goed ik snap je punt. Ik wil wel `_makeRequest()` verwijderen want die functie stopt gewoon met werken. Dus ik schrijf deze twee functies om naar de nieuwe SesClient en geef ze een deprecated vlaggetje.